pkg/cmd/flags/filter.go

```diff
@@ -44,318 +44,63 @@ Multiple values are ORed if used with equals operator '=', but are ANDed if used
-  --filter pidns!=4026531836                                    | only trace events from pidns id not equal to 4026531840
-  --filter tree=476165                                          | only trace events that descend from the process with pid 476165
-  --filter tree!=5023                                           | only trace events if they do not descend from the process with pid 5023
-  --filter tree=3213,5200 --filter tree!=3215                   | only trace events if they descend from 3213 or 5200, but not 3215
-  --filter 'uid>0'                                              | only trace events from uids greater than 0
-  --filter 'pid>0' --filter 'pid<1000'                          | only trace events from pids between 0 and 1000
-  --filter 'u>0' --filter u!=1000                               | only trace events from uids greater than 0 but not 1000
-  --filter event=execve,open                                    | only trace execve and open events
-  --filter event='open*'                                        | only trace events prefixed by "open"
-  --filter event!='open*,dup*'                                  | don't trace events prefixed by "open" or "dup"
-  --filter set=fs                                               | trace all file-system related events
-  --filter s=fs --filter e!=open,openat                         | trace all file-system related events, but not open(at)
-  --filter uts!=ab356bc4dd554                                   | don't trace events from uts name ab356bc4dd554
-  --filter comm=ls                                              | only trace events from ls command
-  --filter binary=/usr/bin/ls                                   | only trace events from /usr/bin/ls binary
-  --filter binary=host:/usr/bin/ls                              | only trace events from /usr/bin/ls binary in the host mount namespace
-  --filter binary=4026532448:/usr/bin/ls                        | only trace events from /usr/bin/ls binary in 4026532448 mount namespace
-  --filter close.args.fd=5                                      | only trace 'close' events that have 'fd' equals 5
-  --filter openat.args.pathname='/tmp*'                         | only trace 'openat' events that have 'pathname' prefixed by /tmp
-  --filter openat.args.pathname='*shadow'                       | only trace 'openat' events that have 'pathname' suffixed by shadow
-  --filter openat.args.pathname!=/tmp/1,/bin/ls                 | don't trace 'openat' events that have 'pathname' equals /tmp/1 or /bin/ls
-  --filter openat.context.processName=ls                        | only trace 'openat' events that have 'processName' equal to 'ls'
-  --filter security_file_open.context.container                 | only trace 'security_file_open' events coming from a container
-  --filter comm=bash --filter follow                            | trace all events that originated from bash or from one of the processes spawned by bash
+Scope examples:
```

Comment:
@yanivagman I kept both `--help event` and `--help scope` showing the same help message since they have a lot in common (operators info above). But let me know if we must split it as well.

Reply:
Event flag only has a subset of the operators, and can have its own help message.
Yet I think it's ok to keep it like that for now.
We can address this in a future PR